ansible/roles/ssc/templates/start.sh

```diff
@@ -23,17 +80,24 @@ if [ ! -f "/root/.ssc/config/genesis.json" ]; then
     sscd init --chain-id "$CHAIN_ID" $MONIKER
   fi
 
-  echo "Downloading genesis file from $GENESIS_URL..."
-  curl -f "$GENESIS_URL" --output /root/.ssc/config/genesis.json || {
-    echo "Failed to download genesis file from $GENESIS_URL"
-    exit 1
-  }
+  if [ -n "$GENESIS_URL" ]; then
+    echo "Downloading genesis file from $GENESIS_URL..."
+    curl -f "$GENESIS_URL" --output /root/.ssc/config/genesis.json || {
+      echo "Failed to download genesis file from $GENESIS_URL"
+      exit 1
+    }
+  fi
 
   echo "Editing config files"
   if [ -n "$PERSISTENT_PEERS" ]; then
     echo "Setting persistent_peers in config.toml..."
     sed -i "s|^persistent_peers *=.*|persistent_peers = \"$PERSISTENT_PEERS\"|" "$CONFIG_FILE"
   fi
+
+  # Configure state sync if enabled
+  if ShouldStartFromStateSync; then
+    ConfigureStartFromStateSync
+  fi
```

Comment:
We want to add the snapshot interval so everyone will serve snapshots by default via p2p. It can be conditional and defaulted to true in the inventory